server/src/server.js

```diff
@@ -0,0 +1,144 @@
+import express from "express";
+import Immutable from 'immutable';
+import moment from 'moment';
+
+let app = express();
+const PORT = 3003;
+
+const ONE_HOUR = 3600000;
+
+let presentData = new Map();
+
+let createMockDateIfNeeded = function createMockDateIfNeeded(applicationId, versionId, startDate, endDate) {
```

Comment:
Why not just `function createMockDateIfNeeded(...)`? Looks better and has basically the same effect.